gradle.properties

```diff
@@ -17,7 +17,7 @@ development=true
 RELEASE_SIGNING_ENABLED=true
 # Library config
 GROUP=com.powersync
-LIBRARY_VERSION=0.0.1-ALPHA17
+LIBRARY_VERSION=0.0.1-BETA1
```

Comment:
If this is a prerelease version. Could this be `1.0.0-BETA1`?

Reply:
Not sure to be honest I just followed the ALPHA pattern. @kobiebotha what are your thoughts?

Reply:
let's go with semver

Reply:
Updated